web/themes/custom/novel/templates/paragraphs/paragraph--material-grid-automatic.html.twig

```diff
@@ -1 +0,0 @@
-{{ react_app_material_grid_automatic }}
```

Comment:
This file (and the others below) were never needed - we just needed to set `$variables['content']['whatever']`, and Drupal would take care of the rest.